WordPress/Classes/Stores/StatsWidgetsStore.swift

```diff
@@ -264,6 +265,10 @@ extension StatsWidgetsStore {
 // MARK: - Login/Logout notifications
 private extension StatsWidgetsStore {
 
+    /**
+     When WPAccountDefaultWordPressComAccountChanged is posted after login
+     the site data is not yet loaded and cannot be cached for widgets.
+     */
```

Comment:
any chance we could use modern-style documentation for methods, using the typical format, like so?

```
/// Store stats in the widget cache
    /// - Parameters:
    ///   - widgetType: concrete type of the widget
    ///   - stats: stats to be stored
```